src/components/search/entry/SortSelect.jsx

```diff
@@ -71,22 +71,29 @@ class SortSelect extends Component {
     } = this.props;
 
     return (
-      <select
-        className={styles.common}
-        value={value}
-        onChange={this.handleChange}
-      >
-        {
-          ['bestmatch', 'atoz', 'ztoa', 'newest', 'oldest'].map((sortOrder) => (
-            <option
-              key={sortOrder}
-              value={sortOrder}
-            >
-              {intl.formatMessage(messages[sortOrder])}
-            </option>
-          ))
-        }
-      </select>
+      <label
+          aria-label="Sort order"
```

Comment:
Remove `aria-label`. `aria-label` is redundant, since you have a `label` with text.

(Also you should have changed it to "Sort by", but it doesn't matter since you're removing it.)